payment/credentials.go

```diff
@@ -88,21 +88,35 @@ func (service *Service) CreateOrderCreds(ctx context.Context, orderID uuid.UUID,
 		return errors.New("order has not yet been paid")
 	}
 
-	issuer, err := service.GetOrCreateIssuer(ctx, order.MerchantID)
-	if err != nil {
-		return errorutils.Wrap(err, "error finding issuer")
-	}
+	// get the order items, need to create issuers based on the
+	// special sku values on the order items
+	for _, orderItem := range order.Items {
+		var issuerID string
+		switch orderItem.SKU {
```

Comment:
would it be better to use a consistent name-spacing scheme for naming the issuers instead of hard coding paths for the anon card / user wallet SKUs?

Reply:
updated in https://github.com/brave-intl/bat-go/pull/303/commits/31dc090cb9957d977ba974ba32c8d791434dbab4